.orchestra/support/revng-container/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM docker.io/library/ubuntu:22.04 as BASE
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+COPY apt.no-extra /etc/apt/apt.conf.d/no-extra
+RUN apt update && \
+    apt install -y \
+        # Needed for recompilation pipe (-dev included due to linking problems)
+        libc6 libc6-dev \
+        # Needed for revng.cli etc.
+        python3 ca-certificates \
+        # some LLVM libs are not searched properly
+        libllvm12 libc++1-12 libatomic1 && \
+    apt upgrade -y && \
+    rm -rf /var/lib/apt/lists/* /var/log/*
+
+# For some reason c++ does not look for librt.so.1 but only for librt.so
+RUN cd /lib/x86_64-linux-gnu && \
+    for file in *.so.*; do \
+        SO_FILE=$(echo $file | grep -Po '^.*\.so'); \
+        if [ ! -e $SO_FILE ]; then ln -s $file $SO_FILE; fi \
+    done
```

Comment:
These are all hacks that will need a more proper fix. Let's do only the strictly necessary set of hacks, so just symlink `librt.so` not all the libs.

Reply:
Ran a round of strace, seems the only library affected was `librt.so`, symlinked only that one